translations/de.php

```diff
@@ -140,11 +166,59 @@
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_d22e8bf6263dfeb9e56945a2075b6b34'] = 'Bezahlen mit ';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_644818852b4dd8cf9da73543e30f045a'] = 'Zurück zur Bezahlung';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_6ff063fbc860a79759a7369ac32cee22'] = '';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_21ff087f1520a70163128cae982de67a'] = 'Kartenzahlung';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_f1d3b424cd68795ecaa552883759aceb'] = 'Bestellübersicht';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_879f6b8877752685a966564d072f498f'] = 'Ihr Einkaufswagen ist leer.';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_1d565b9e5303987bb1b1938d5d458bca'] = 'Gutschein';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_ad69e733ebae8d264bccaa38d68830e8'] = 'PayPal';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_18356e0e1a570420945a00fac60fae68'] = 'Sie haben sich dafür entschieden, mit Karte zu zahlen.';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_7253e56eef5057e5763f535498a4abe2'] = 'Sie haben sich dafür entschieden, mit PayPal zu zahlen.';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_c884ed19483d45970c5bf23a681e2dd2'] = 'Hier ist eine kurze Zusammenfassung Ihrer Bestellung:';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_32231b6aa7f7da0da19a691bd78c5680'] = 'Der Gesamtbetrag Ihrer Bestellung beläuft sich auf';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_edd87c9059d88fea45c0bd6384ce92b9'] = 'Bitte bestätigen Sie Ihre Bestellung, indem Sie auf „Ich bestätige meine Bestellung“ klicken.';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_a44217022190f5734b2f72ba1e4f8a79'] = 'Kartennummer';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_95b16127e70e8a90220404fb48343182'] = 'Ablaufdatum';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_16aefbf48d8803216285343472025aed'] = 'MM/YY';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_5ea7f231296949070013bc43715aae01'] = 'CVC';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_bc9189406be84ec297464a514221406d'] = 'XXX';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_e81c4e4f2b7b93b481e13a8553c2ae1b'] = 'oder';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_2e3e721e530a957f81d5b5dcff0bf0e1'] = 'Schneller Checkout';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_f2e984154b51f09b97cd60a1b8396059'] = '';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_569fd05bdafa1712c4f6be5b153b8418'] = 'Weitere Zahlungsmethoden';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_46b9e3665f187c739c55983f757ccda0'] = 'Ich bestätige meine Bestellung';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_100e10b5e05d4b3abd2cd52fd12e8789'] = 'Bei der Bezahlung ist ein Fehler aufgetreten. Bitte versuchen Sie es erneut oder wenden Sie sich an das Support-Team.';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_72e506c5e5a9308bb3836d539dc4e39e'] = '';
+$_MODULE['<{ps_checkout}prestashop>ps_checkout_490098b6f04cb77f81f6e4878f6869f2'] = '';
```

Comment:
I found several times empty strings in the file, I'm not sure this should happen (Content would be empty in this language)